topics/proteomics/tutorials/peptide-library-data-analysis/tutorial.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: tutorial_hands_on
+
+title: Peptide Library Data Analysis
+questions:
+  - How to utilize quantitative properties of amino acids and peptide sequence to analyse peptide data?
+objectives:
+  - Calculate descriptors
+  - Quantitative analysis of peptide sequence properties
+time_estimation: '20m'
+contributors:
+   - jaidevjoshi83
+   - blankenberg
+---
+
+# Introduction
+{:.no_toc}
+
+<!-- This is a comment. -->
+
+Several computational methods have been proven very useful in the initial screening and prediction of peptides for various biological properties. These methods have emerged as effective alternatives to the lengthy and expensive traditional experimental approaches. In this tutorial, we will be discussing how to analyze peptide libraries on the basis of quantitative properties. In this tutorial, we will learn how to use different utilities of PDAUG to calculate various peptide-based features and utilize these features for various informative plots.
+
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+
+## Peptide Data
+
+In this step, we will retrieve the inbuild dataset, which contains anti-microbial (AMPs) and transmembrane peptides (TMPs).
+
+
+> ### {% icon hands_on %} Hands-on: Task description
+>
+> 1. **PDAUG Peptide Data Access** {% icon tool %} with the following parameters:
+>    - *"Datasets"*: `AMPvsTM` 
+>
+{: .hands_on}
+
+
+## Converting tabular data into fasta format
+
+In this step, we will be converting and splitting the tabular data file into fasta format. This tool splits data into two files based on their class labels.
+
+> ### {% icon hands_on %} Hands-on: Task description
+>
+> 1. **PDAUG TSVtoFASTA** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Input file"*: `PDAUG Peptide Data Access - AMPvsTM (tabular)` (output of **PDAUG Peptide Data Access** {% icon tool %})
+>    - *"Data conversion"*: ` WithClassLabel `
+>
+>
+{: .hands_on}
+
+## Analyzing peptide libraries (AMPs and TMPs) based on features and feature space 
+
+### Summary Plot for peptide libraries 
+
+In this step, we utilize **PDAUG Peptide Sequence Analysis** tool to compare peptide sequences based on hydrophobicity, hydrophobic movement, charge, amino acid fraction, and sequence length and creates a summary plot.
+
+
+
+> ### {% icon hands_on %} Hands-on: Task description
+>
+> 1. **PDAUG Peptide Sequence Analysis** {% icon tool %} with the following parameters:
```

Comment:
@jaidevjoshi83 can you use the new fancy tool annotation.
e.g. like `> 1. {% tool [PDAUG Peptide Sequence Analysis](https://toolshed.g2.bx.psu.edu/view/galaxyp/metaquantome_sample/49a957fb353a) %} with the following parameters:` as seen here: https://raw.githubusercontent.com/galaxyproject/training-material/2c43dbc2093be05106cecdc20dd7daa348d2180f/topics/proteomics/tutorials/metaquantome-taxonomy/tutorial.md

Reply:
https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#tool-links is a good link for that.

Reply:
New fancy tool annotation added